integration/benchnet2/automate/bootstrap_automation_test.go

```diff
@@ -0,0 +1,55 @@
+package automate
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestGenerateTestTemplates(t *testing.T) {
+	fmt.Printf("Starting tests")
+	expectedValues := textReader("templates/test_templates/expected_values.yml")
```

Comment:
Isn't this using `textReader` to calculate the expected values? But aren't we testing `textReader`? How can we trust that the results are correct from `textReader`?